docs/Chipyard-Basics/Initial-Repo-Setup.rst

```diff
@@ -1,6 +1,21 @@
 Initial Repository Setup
 ========================================================
 
+Requirements
+-------------------------------------------
+
+Using Linux is recommended.
```

Comment:
I would reword to... "Chipyard developed and tested on Linux based systems." That emphasizes that we haven't thought of macOS support and don't really care about it.